doc/cephfs/dirfrags.rst

```diff
@@ -46,8 +46,8 @@ Size thresholds
 ===============
 
 A directory fragment is eligible for splitting when its size exceeds
-``mds_bal_split_size`` (default 10000).  Ordinarily this split is
-delayed by ``mds_bal_fragment_interval``, but if the fragment size
+``mds_bal_split_size`` (default 10000 directory entries).  Ordinarily this
+split is delayed by ``mds_bal_fragment_interval``, but if the fragment size
```

Comment:
To be clear, the split is *always* delayed by the interval, even if there hasn't been any current/recent activity?

Reply:
Yes, even if there is no IO to be taken care of i.e. no current/recent activity, MDS will still leave a window of the interval(delay) mentioned.

Reply:
I think this info should be included in this doc. I'll try add and push it in sometime.